library/src/main/java/libxmtp-version.txt

```diff
@@ -1,3 +1,3 @@
-Version: c2b99064
+Version: e1bbb348
 Branch: main
-Date: 2025-03-06 17:48:33 +0000
+Date: 2025-03-08 01:05:24 +0000
```

Comment:
This is the release candidate from Friday.

Reply:
But sounds like we're going to make another one today with the web fix so we can bump again.